.github/workflows/preview-build.yml

```diff
@@ -44,6 +51,37 @@ permissions:
   pull-requests: read
 
 jobs:
+  match:
+    if: github.event.repository.fork == false # Skip running the job on the fork itself (It still runs on PRs on the upstream from forks)
+    concurrency:
+      group: ${{ github.workflow }}-${{ github.event.pull_request.head.ref || github.ref }}
+      cancel-in-progress: ${{ startsWith(github.event_name, 'pull_request') }}
+    runs-on: ubuntu-latest
+    outputs:
+      content-source-match: ${{ steps.event-check.outputs.content-source-match != '' && steps.event-check.outputs.content-source-match || steps.match.outputs.content-source-match }}
+      content-source-name: ${{ steps.event-check.outputs.content-source-name != '' && steps.event-check.outputs.content-source-name || steps.match.outputs.content-source-name }}
+    steps:
+      - name: Not a push event
+        id: event-check
+        if: contains(fromJSON('["merge_group", "pull_request", "pull_request_target"]'), github.event_name)
+        run: |
+          echo "content-source-match=true >> $GITHUB_OUTPUT
+          echo "content-source-name=next >> $GITHUB_OUTPUT
```

Comment:
IIUC, these are just some "random" values so that the build runs on PRs.

Can we add a comment saying that. (Assuming my assumption is true)